runtime/src/accounts_db.rs

```diff
@@ -2454,6 +2589,7 @@ impl AccountsDB {
             // It should not be possible that a slot is neither in the cache or storage. Even in
             // a slot with all ticks, `Bank::new_from_parent()` immediately stores some sysvars
             // on bank creation.
+            self.uncleaned_keys.remove(remove_slot);
```

Comment:
nits: how about adding blank line above to make it clear the above comment isn't explaining this line?